Dockerfile

```diff
@@ -1,20 +1,21 @@
 FROM postgres:9.5
 
-ENV RUST_VERSION 1.8.0
 ENV PATH ~/.cargo/bin/:$PATH
 
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends ca-certificates curl git make gcc postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip \
-  && mkdir -p /tmp/build \
-  && curl -o /tmp/build/rust-${RUST_VERSION}.tar.gz -SL https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
-  && tar -xzf /tmp/build/rust-${RUST_VERSION}.tar.gz -C /tmp/build/ \
-  && sh /tmp/build/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu/install.sh \
+  && apt-get install -y --no-install-recommends \
+  ca-certificates curl git make gcc postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip \
```

Comment:
It's really nothing so you can ignore it :) I just meant adding two spaces before `ca-certificates` to avoid aligning with `&&`, otherwise it may look like another command.